build-gcc.sh

```diff
@@ -16,8 +16,8 @@ done
 # TODO: Better target handling
 case "${arch}" in
   "arm") TARGET="arm-eabi" ;;
-  "arm64") TARGET="aarch64-elf" ;;
-  "x86") TARGET="x86_64-elf" ;;
+  "arm64") TARGET="aarch64-linux-gnu" ;;
```

Comment:
I'd rather not, Eva GCC was majorly focused to be bare metal. You can add another option as arm64-full.

Reply:
> I'd rather not, Eva GCC was majorly focused to be bare metal. You can add another option as arm64-full.

for eva upto a486e3e